.gitlab-ci.yml

```diff
@@ -107,6 +107,7 @@ Base:
           - aws/centos-stream-8-aarch64
       - RUNNER:
           - aws/rhel-8.4-x86_64
+          - aws/rhel-8.5-x86_64
```

Comment:
FTR this is going to fail b/c of the other missing images. Once the CI job passes we should vet the tests which failed, there should be only a few of them which either use qcow2 or try to compare the list of available image types. Everything else should report PASS.

Reply:
What's the plan then?  Are we keeping this open until all the other image types land?  I don't mind either way.

Reply:
I guess the plan is up to you ;) Would it be a lot of churn to only enable the tests we expect to pass for this PR, and then enable the rest when the other image types are ready?

Reply:
@achilleas-k let's not keep PRs open longer than they need to. The fact that there's missing support for image types (which has the nice side effect of breaking our test suite) is not related to the work around tar installer.